<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>600</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pg_semafor">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lb_semafor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Semáforo</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="lb_vermell">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="lb_groc">
          <property name="minimumSize">
           <size>
            <width>49</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>49</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="lb_verd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_cicle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Circle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_cicle"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_configurar">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lb_velocitat">
          <property name="text">
           <string>Velocidad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDial" name="dial_velocitat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_ok1">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_quant_a">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Semàfors&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Visualitzacio de cicles semafòrics.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;2021. Institut Marianao.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_ok2">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
    </font>
   </property>
   <widget class="QMenu" name="menuFichero">
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
    <property name="title">
     <string>Fichero</string>
    </property>
    <addaction name="ac_configurar"/>
    <addaction name="ac_sortir"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="ac_quant_a"/>
   </widget>
   <addaction name="menuFichero"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Cascadia Code</family>
    </font>
   </property>
  </widget>
  <action name="ac_configurar">
   <property name="text">
    <string>Configurar</string>
   </property>
  </action>
  <action name="ac_sortir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="ac_quant_a">
   <property name="text">
    <string>Cuanto a</string>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
